src/utils/i18n.js

```diff
@@ -0,0 +1,13 @@
+import i18n from "i18n";
+import path from 'path';
+
+const __dirname = path.resolve();
+
+i18n.configure({
+	locales: ["en", "kn", "fr"],
+	defaultLocale: "en",
+	directory: `${__dirname}/locales/`,
+	headers: "Accept-Language"
+});
+
+export default i18n;
```

Comment:
for this configuration of i18n make sure we are using the same folder for all configurations to keep a consistent project structure pick a folder and use all configs there

Reply:
noted, thanks